### PR TITLE
Py3 tutorial

### PR DESCRIPTION
**Pull request by ankostis**
Make tutorial-code py3 compatible.

- Fix print-statement.
- Drop {}.iterkeys().
- STILL left one `iteritems()` (doc/tutorial/cproject.py)

### CODE REVIEW

File: doc/tutorial/clean_mix.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 from doit.task import clean_targets
 
 def simple():
-    print "ok"
```
Comment (schettino72) [inline doc/tutorial/clean_mix.py]: it seems you removed all quotes `"`

Comment (ankostis) [inline doc/tutorial/clean_mix.py]: Crafting untested commits from regexes late at night will not work as intended :-(
Wil fix them manually.

Comment (saimn) [inline doc/tutorial/clean_mix.py]: python-modernize is quite handy for this, for example for prints you can use this:
```
python-modernize -w -f libmodernize.fixes.fix_print dodo.py
```
http://python-modernize.readthedocs.org/en/latest/

Comment (ankostis) [inline doc/tutorial/clean_mix.py]: great thanks!

**Comment by ankostis** [comment]
Take 2: Used `future` modernize with hand-removing needless changes.

Left ``from __future import print`` statement in case the tutorial-code gets test-cased ever.

**Comment by schettino72** [comment]
thanks!

**Comment by Kwpolska** [comment]
In most cases, you could leave the __future__ import out, provided that you don’t use commas.

```pycon
>>> #py2
>>> print("hello")
hello
>>> print("hello", "world")
("hello", "world")
```

```pycon
>>> #py3
>>> print("hello")
hello
>>> print("hello", "world")
hello world
```

**Pull request merged**